jvcprojector/projector.py

```diff
@@ -101,50 +100,140 @@ async def disconnect(self) -> None:
 
     async def get_info(self) -> dict[str, str]:
         """Get device info."""
-        assert self._device
-        model = JvcCommand(command.MODEL, True)
-        mac = JvcCommand(command.MAC, True)
-        await self._send([model, mac])
+        if not self._device:
+            raise JvcProjectorError("Must call connect before getting info")
+
+        model = JvcCommand(const.CMD_MODEL, True)
+        mac = JvcCommand(const.CMD_LAN_SETUP_MAC_ADDRESS, True)
+        version = JvcCommand(const.CMD_VERSION, True)
```

Comment:
CMD_VERSION (IFSV) is timing out 100% of the time when my projector is not in ON (in STANDBY or COOLING). I tried bumping up the sleep between commands but made no difference.

```
2024-09-23 11:01:32.103 DEBUG (MainThread) [jvcprojector.device] Connecting to 192.168.0.30
2024-09-23 11:01:32.116 DEBUG (MainThread) [jvcprojector.device] Handshake received b'PJ_OK'
2024-09-23 11:01:32.118 DEBUG (MainThread) [jvcprojector.device] Handshake sending 'PJREQ'
2024-09-23 11:01:32.120 DEBUG (MainThread) [jvcprojector.device] Handshake received b'PJACK'
2024-09-23 11:01:32.120 DEBUG (MainThread) [jvcprojector.device] Sending op ' (b'!\x89\x01\x00\x00\n')'
2024-09-23 11:01:32.121 DEBUG (MainThread) [jvcprojector.device] Received ack b'\x06\x89\x01\x00\x00\n'
2024-09-23 11:01:32.322 DEBUG (MainThread) [jvcprojector.device] Sending ref 'MD (b'?\x89\x01MD\n')'
2024-09-23 11:01:32.325 DEBUG (MainThread) [jvcprojector.device] Received ack b'\x06\x89\x01MD\n'
2024-09-23 11:01:32.425 DEBUG (MainThread) [jvcprojector.device] Received ref b'ILAFPJ -- B2A2' (b'@\x89\x01MDILAFPJ -- B2A2\n')
2024-09-23 11:01:32.626 DEBUG (MainThread) [jvcprojector.device] Sending ref 'LSMA (b'?\x89\x01LSMA\n')'
2024-09-23 11:01:32.633 DEBUG (MainThread) [jvcprojector.device] Received ack b'\x06\x89\x01LS\n'
2024-09-23 11:01:32.733 DEBUG (MainThread) [jvcprojector.device] Received ref b'E0DADC0A1562' (b'@\x89\x01LSE0DADC0A1562\n')
2024-09-23 11:01:32.934 DEBUG (MainThread) [jvcprojector.device] Sending ref 'IFSV (b'?\x89\x01IFSV\n')'
(long delay)
2024-09-23 11:01:47.937 WARNING (MainThread) [jvcprojector.device] Response timeout for 'IFSV'
2024-09-23 11:01:48.139 DEBUG (MainThread) [jvcprojector.device] Disconnected
2024-09-23 11:01:48.139 ERROR (MainThread) [homeassistant.components.jvc_projector.coordinator] Unexpected error fetching JVC Projector data
Traceback (most recent call last):
  File "/Volumes/code/python/home-assistant/homeassistant/helpers/update_coordinator.py", line 354, in _async_refresh
    self.data = await self._async_update_data()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Volumes/code/python/home-assistant/homeassistant/components/jvc_projector/coordinator.py", line 47, in _async_update_data
    state = await self.device.get_state()
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Volumes/code/python/iloveicedgreentea/pyjvcprojector/jvcprojector/projector.py", line 134, in get_state
    self._dict[const.KEY_VERSION] = self.process_version(self._version)
                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Volumes/code/python/iloveicedgreentea/pyjvcprojector/jvcprojector/projector.py", line 213, in process_version
    version = version.removesuffix("PJ")
              ^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'removesuffix'
2024-09-23 11:01:48.141 DEBUG (MainThread) [homeassistant.components.jvc_projector.coordinator] Finished fetching JVC Projector data in 0.002 seconds (success: False)
```

Reply:
added a check for power state